canonical-web-and-design/canonical.com
# Add filter and search to find-a-partner page

## Done
- Add search functionality to `find-a-partner` page
- Add filter functionality to `find-a-partner` page


## QA
- Run the website locally using `./run`
- Go to [/partners/find-a-partner](http://0.0.0.0:8002/partners/find-a-partner) and search for partners and filter partners 
- Make sure it behaves similar to https://partners.ubuntu.com/find-a-partner


## Issue
Fixes https://github.com/canonical-web-and-design/web-squad/issues/1645
